packages/ses/package.json

```diff
@@ -91,6 +101,7 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.29.1",
+    "hermes-engine-cli": "0.12.0",
```

Comment:
nb: after few yrs and [requesting](https://github.com/facebook/hermes/issues/1371#issuecomment-2052072367) old Hermes Engine CLI be updated - to avoid later extending CI to [build and run](https://github.com/facebook/hermes/blob/main/doc/BuildingAndRunning.md) later versions of Hermes (to support later versions of React Native) - we finally have it landed 🎉 

https://github.com/facebook/hermes/releases/tag/v0.13.0

Reply:
however not yet [published](https://www.npmjs.com/package/hermes-engine-cli) on npm, so remains to be implemented later the [previous](https://github.com/endojs/endo/pull/2334/commits/0b1ca82f82ced36a65f6fde5dd8ff07c45043427) way
(fetching then unpacking the .tar.gz)